api/v1alpha1/olsconfig_types.go

```diff
@@ -102,6 +102,9 @@ type OLSSpec struct {
 	// User data collection switches
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="User Data Collection"
 	UserDataCollection UserDataCollectionSpec `json:"userDataCollection,omitempty"`
+	// extra certs - list of paths
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Extra Certs"
+	ExtraCerts []string `json:"extraCerts,omitempty"`
```

Comment:
We should put either name of configmaps/secrets here, or the content of the certificates. 
The file path as well as the app server config file should be generated by the operator at runtime.

1. If we put into this field the certificate itsself, operator creates the configmap holding these certificates, mount it to the app server deployment and add mouted path to the config file. 
2. If we put a configmap/secret name, we have these options:
    A. assume the certificate key name in the configmap/secret 
    B. we assume all the keys in the configmap/secret is certificates
    C. Add another field for the key in the configmap/secret to locate the certificate.